test/e2e/common/secrets.go

```diff
@@ -313,3 +317,80 @@ func doSecretE2EWithMapping(f *framework.Framework, mode *int32) {
 
 	f.TestContainerOutput("consume secrets", pod, 0, expectedOutput)
 }
+
+//Existence of a second secret with same name and different namespace shouldn't affect the mount
+func doSecretE2EForMountNameSpaceIssue(f *framework.Framework, defaultMode *int32) {
```

Comment:
It is used in the match with expected output.
Also, just wanted to keep it consistent with the other tests.